.storybook/theme/ThemeProvider.js

```diff
@@ -53,6 +53,12 @@ const theme = {
     xl: '1.5rem',
     xxl: '2rem',
   },
+  lineHeights: {
+    s: '1rem',
+    m: '1.4rem',
+    l: '1.5rem',
```

Comment:
How come 1.4 for M and 1.5 for L - should it not be somewhere closer to the center like 1.25 for M?

Reply:
I followed the actual values from `styles.css`
https://github.com/heetch/flamingo/blob/master/src/styles.css#L44-L47